napari/_qt/qt_event_loop.py

```diff
@@ -133,6 +134,16 @@ def get_app(
         app.setWindowIcon(QIcon(kwargs.get('icon')))
         set_app_id(kwargs.get('app_id'))
 
+        if os.getenv("NAPARI_CATCH_ERRORS") not in ('0', 'False'):
+
+            def loc(*args, **kwargs):
+                NapariNotification.show_notification(*args, **kwargs)
+                import gc
+
+                gc.collect()
```

Comment:
can you try:

```
export DEBUG_NAPARI_NOTIFICATION=1
napari
```

That for me make napari crash; of course the fact that GC is called here is random but I'm hopping to not be the only one that get a crash.